documentation/c01-whatis.sil

```diff
@@ -82,11 +82,105 @@ SILE does not benefit from the large ecosystem and community that has grown up a
 In that sense, TeX will remain ahead of SILE for some time to come.
 But in terms of \em{core capabilities}, SILE aims at being at least equivalent to TeX.
 
+\section{SILE versus Typst and other new alternatives}
+
+Almost exactly one decade after SILE development started, a new competitor to LaTeX entered the scene with a bang:
+\footnote{Days of front-page coverage on Hacker News, Lobsters, and the like propelled it past 10k GitHub stars in the first 6 days of being public, a milestone SILE is still very far from reaching.}
+Typst.
+There is no shame in stating that Typst is a great piece of software and a worthy competitor to SILE as well as well as LaTeX.
+Typst is (very) fast, well documented, and relatively versatile.
+While both typesetting engines cover much of the same ground, the respective focus and implementation choices of each are so varied that direct comparisons don’t always have clear winners.
+At the end of the day the specific use case and workflow preferences are at least as relevant as the overall merits of each engine.
+
+When Typst was first conceived, SILE had been around the block in production for a while; but had only really been used for prose.
```

Comment:
"in production for a while"? It's hard to find any sample. 

"used for prose" - Interesting. Clarify "prose" (an no, it's not only about maths being, or not being, in the picture).